wordpress-mobile/WordPress-iOS
# Bump iOS deployment target from 13.0 to 14.0

Bump iOS deployment target from 13.0 to 14.0.

## To test

The app should build locally. Ideally, there should be no deprecation warnings or errors.

~~Currently, we have a bunch of errors due to "Today View extensions have been deprecated". The reason we have an error and not a warning is because those targets have "Treat Warnings as Errors" set to `YES`.~~ Update: #19509 addressed those. 

<details>
<summary>See Today View errors</summary>

![Untitled](https://user-images.githubusercontent.com/1218433/197158238-aec9626b-5b69-4703-8e28-3cc97ddd3f18.jpg)

</details>

## Regression Notes

1. Potential unintended areas of impact – N.A.
2. What I did to test those areas of impact (or what existing automated tests I relied on) – N.A.
3. What automated tests I added (or what prevented me from doing so) – N.A.

---
